src/test/java/com/redhat/devtools/lsp4ij/ui/IconMapperTest.java

```diff
@@ -13,20 +13,73 @@
  *******************************************************************************/
 package com.redhat.devtools.lsp4ij.ui;
 
+import com.intellij.icons.AllIcons;
+import com.intellij.util.ui.ColorIcon;
+import org.eclipse.lsp4j.CompletionItem;
 import org.eclipse.lsp4j.CompletionItemKind;
+import org.eclipse.lsp4j.MarkupContent;
 import org.junit.Test;
 
+import javax.swing.*;
+
 import static com.redhat.devtools.lsp4ij.ui.IconMapper.getIcon;
+import static org.junit.jupiter.api.Assertions.*;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertNull;
 
 public class IconMapperTest {
 
     @Test
     public void getIconTest()  {
-        assertNull(getIcon(null));
+        assertNull(getIcon((CompletionItemKind)null));
         for (CompletionItemKind value : CompletionItemKind.values()) {
             assertNotNull(getIcon(value), "Missing matching icon for "+value);
         }
     }
+
+    @Test
+    public void getColorIconTest()  {
+        String color = "#FFFFFF";
```

Comment:
Add a test with String color = "red";

Reply:
won't work:
- "red" doesn't start with #
- Color.decode(key) will fail.

Is there a "color name to hex" API somewhere?

Reply:
Test are hard to read, could you split the test in several methods please.